src/libcore/fmt/num.rs

```diff
@@ -79,7 +84,9 @@ trait GenericRadix {
                 x = x / base;                  // Deaccumulate the number.
                 *byte = self.digit(n.to_u8()); // Store the digit in the buffer.
                 curr -= 1;
-                if x == zero { break };        // No more digits left to accumulate.
+                if x == zero {
+                    break
+                };        // No more digits left to accumulate.
```

Comment:
Did you manually fix the above change but not this one? If not then it is probably a weird bug in rustfmt. In any case, could you fixup this comment too please?